tests/functional/base_pages/feedback-sent.page.js

```diff
@@ -0,0 +1,12 @@
+import QuestionPage from "./question.page";
+
+class FeedbackSentPage extends QuestionPage {
```

Comment:
This should extend from a new `FeedbackBasePage` as if you call `url()` from a feedback page it will be incorrect.